metaflow/cli.py

```diff
@@ -974,6 +974,13 @@ def start(
     ctx.obj.monitor.start()
     _system_monitor.init_system_monitor(ctx.obj.flow.name, ctx.obj.monitor)
 
+    current._update_env(
+        {
+            "system_logger": _system_logger,
```

Comment:
can you help me with an example of how this is consumed? happy to chat on slack too.